eng/checkout-job.yml

```diff
@@ -0,0 +1,60 @@
+### Check out job pulls down the GIT repository and packs its contents
+### into an Azure artifact for reuse by the subsequent build and test execution phases.
```

Comment:
Do you mind adding a comment, explaining that we have to split between Unix and Windows simple because the compression is OS specific.

Reply:
Good point, added in 2nd commit.